static/js/components/GcnSelectionForm.jsx

```diff
@@ -190,6 +190,11 @@ const GcnSelectionForm = ({ gcnEvent }) => {
     setIsSubmitting(false);
   };
 
+  // const handleSendToObservationPlan = (instrument) => {
```

Comment:
Can you bring this back?

Reply:
Can it just stay commented until there's some sort of request being made? the handler doesn't do anything right now.

Reply:
Ah of course!